01-js-intro-and-jquery/index.js

```diff
@@ -0,0 +1,46 @@
+$(document).ready(function (){
+	$('#sec').hide();
```

Comment:
This is ok, but while the document is not ready, the section will be visible, an alternative would be hiding it with css.